js/stats.js

```diff
@@ -34,7 +34,7 @@ function displayStatsJSON (reportText) {
   div.innerHTML = (
     (stats.n_repos / 1e3).toFixed(1) + 'k feedstocks<br>' +
     (stats.n_packages / 1e3).toFixed(1) + 'k packages<br>' +
-    (stats.n_artifacts / 1e3).toFixed(0) + 'k artifacts'
+    (stats.n_artifacts / 1e6).toFixed(1) + 'M artifacts'
```

Comment:
Is there a way to automate bumping this? Perhaps some JavaScript package that renders numbers in scientific format or similar?

Reply:
This is how it looks like now:

![image](https://user-images.githubusercontent.com/2559438/187391952-4df1f5f1-1a67-4650-8c7c-ded3a3711c73.png)


I don't anticipate another change anytime soon, but I guess we could have a little function to automate the formatting.

Reply:
e.g. like this https://stackoverflow.com/a/60550008/3407590